atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1139,7 +1157,17 @@ void WebContents::Print(mate::Arguments* args) {
   printing::PrintViewManagerBasic::FromWebContents(web_contents())->
       PrintNow(web_contents()->GetMainFrame(),
                settings.silent,
-               settings.print_background);
+               settings.print_background,
+               settings.device_name);
+}
+
+
+std::vector<printing::PrinterBasicInfo> WebContents::GetPrinterList(
+    mate::Arguments* args) {
+  std::vector<printing::PrinterBasicInfo> printerList;
```

Comment:
Usually underscores are used instead of camel case for variable names. So perhaps this should be `printer_list` or just `printers`. Same thing for `printBackend` below.